keylime/tpm/tpm_main.py

```diff
@@ -849,15 +850,15 @@ def quoterepl(m):
             vendorStr = retyaml["TPM2_PT_VENDOR_STRING_1"]["value"]
         elif "TPM_PT_VENDOR_STRING_1" in retyaml:
             vendorStr = retyaml["TPM_PT_VENDOR_STRING_1"]["as string"].strip()
-        else:
-            vendorStr = ""
```

Comment:
I added this previously but it's not necessary if we change the return value to `Optional[str]`